pkg/sql/colfetcher/cfetcher.go

```diff
@@ -884,8 +884,10 @@ func (rf *cFetcher) pushState(state fetcherState) {
 
 // getDatumAt returns the converted datum object at the given (colIdx, rowIdx).
 // This function is meant for tracing and should not be used in hot paths.
-func (rf *cFetcher) getDatumAt(colIdx int, rowIdx int, typ *types.T) tree.Datum {
-	return colexec.PhysicalTypeColElemToDatum(rf.machine.colvecs[colIdx], rowIdx, &rf.table.da, typ)
+func (rf *cFetcher) getDatumAt(colIdx int, rowIdx int) tree.Datum {
+	res := []tree.Datum{nil}
```

Comment:
Does this cause an allocation? I don't think so but if it did that would be bad news.